src/Microsoft.ML.Transforms/RffTransform.cs

```diff
@@ -39,9 +40,10 @@ public sealed class Arguments
             [Argument(ArgumentType.AtMostOnce, HelpText = "The number of random Fourier features to create", ShortName = "dim")]
             public int NewDim = Defaults.NewDim;
 
-            [Argument(ArgumentType.Multiple, HelpText = "which kernel to use?", ShortName = "kernel")]
-            public SubComponent<IFourierDistributionSampler, SignatureFourierDistributionSampler> MatrixGenerator =
-                new SubComponent<IFourierDistributionSampler, SignatureFourierDistributionSampler>(GaussianFourierSampler.LoadName);
+            [Argument(ArgumentType.Multiple, HelpText = "which kernel to use?", ShortName = "kernel", SignatureType = typeof(SignatureFourierDistributionSampler))]
```

Comment:
>w [](start = 57, length = 1)

Not your fault, but since you touch it, can we capitalize it? Here and in for UseSin field?